packages/pn-personafisica-webapp/public/locales/it/recapiti.json

```diff
@@ -133,13 +133,14 @@
     }
   },
   "courtesy-contacts": {
```

Comment:
Check localizations and remove those that are not used anymore from other languages